domFoundations/events,storage,attributes/main.js

```diff
@@ -0,0 +1,19 @@
+'use strict'
+
+import { securedTrigSwitch } from "./trigBox.js";
+import { trigCheck } from "./trigBox.js";
+import { buttonListener } from "./buttonListener.js";
+import { lightSwitcher } from "./lightSwitcher.js";
+import { textSwitcher } from "./textSwitcher.js";
+import { renderDate } from "./dateRenderer.js";
+import { startStateRenderer } from "./initialPageRenderer.js";
+
+const startAppFunc = () => {
+    console.log ('App started');
+
+    startStateRenderer();
+      
+    buttonListener(lightSwitcher.bind(this, trigCheck), textSwitcher.bind(this, trigCheck), securedTrigSwitch, renderDate);
```

Comment:
it's very hard to understand what happens here. Change it to more readable (do not use `.bind`)

Reply:
Please clarify if there are other ways to pass an argument to the function without calling it. I use `.bind` to pass the arguments.